src/main/kotlin/ua/marchenko/artauction/artwork/mapper/ArtworkMapperExtension.kt

```diff
@@ -0,0 +1,22 @@
+package ua.marchenko.artauction.artwork.mapper
+
+import ua.marchenko.artauction.artwork.controller.dto.CreateArtworkRequest
+import ua.marchenko.artauction.artwork.controller.dto.ArtworkResponse
+import ua.marchenko.artauction.artwork.enums.ArtworkStatus
+import ua.marchenko.artauction.artwork.enums.ArtworkStyle
+import ua.marchenko.artauction.artwork.model.Artwork
+import ua.marchenko.artauction.user.mapper.toUserResponse
+import ua.marchenko.artauction.user.model.User
+
+fun Artwork.toArtworkResponse() = ArtworkResponse(
+    id?.toString() ?: "unknown",
```

Comment:
```suggestion
    id!!.toHexString(),
```
usually hex representation of ObjectId is used

Reply:
done